Cargo.lock

```diff
@@ -1244,14 +1244,14 @@ version = "0.3.0"
 dependencies = [
  "libc",
  "utils",
- "vm-memory 0.8.0",
+ "vm-memory 0.9.0",
```

Comment:
Shouldn't we also bump the internal dependency too ([here](https://github.com/firecracker-microvm/firecracker/blob/00ebfe26e97258b6f489ab38c47057a38aef7118/src/vm-memory/Cargo.toml#L10))?

Reply:
Ok, updating the internal dependency to 0.9.0 as well.